client/dashboard/task-list/index.js

```diff
@@ -256,6 +256,58 @@ class TaskDashboard extends Component {
 		);
 	}
 
+	onSkipStoreSetup = () => {
+		const completedTasksNames = this.getTasks()
+			.filter( x => x.completed )
+			.map( x => x.title );
```

Comment:
Let's use the `key` here in case the copy for titles changes in the future.

Reply:
Done